controllers/rabbitmqcluster_controller.go

```diff
@@ -101,6 +102,16 @@ func (r *RabbitmqClusterReconciler) Reconcile(req ctrl.Request) (ctrl.Result, er
 		return ctrl.Result{}, r.prepareForDeletion(ctx, rabbitmqCluster)
 	}
 
+	// exit if skip reconcile annotation is set to true
+	if v, ok := rabbitmqCluster.Annotations["rabbitmq.com/skipReconcile"]; ok && v == "true" {
```

Comment:
Let's use the constant defined above.

Reply:
rabbitmqCluster.Annotations need to avoid go panic.